### PR TITLE
Add BYO notebook

### PR DESCRIPTION
**Pull request by youran-qi**
Copy the BYO notebook from https://github.com/cohere-ai/cohere-aws/blob/main/notebooks/sagemaker/Deploy%20your%20own%20finetuned%20command-r-0824.ipynb to here

### CODE REVIEW

**Comment by kyle-cohere** [comment]
Can you please replace the title header with an h2 instead of h1? (Use `##` instead of `#`). The compiler will think it's a python comment

**Comment by youran-qi** [comment]
> Can you please replace the title header with an h2 instead of h1? (Use `##` instead of `#`). The compiler will think it's a python comment

OK. @kyle-cohere Done. See [here](https://github.com/cohere-ai/notebooks/pull/222/commits/f2cb81e58e82a608bc41aa5edc340ef980dd3594)

File: notebooks/finetuning/Deploy your own finetuned command-r-0824.ipynb
Comment (CoderHam) [inline notebooks/finetuning/Deploy your own finetuned command-r-0824.ipynb:83]:
```suggestion
    "!pip install \"cohere>=5.11.0\""
```

**Review by CoderHam — CHANGES_REQUESTED** [review]
Minor change else LGTM
Please test before merge?

File: notebooks/finetuning/Deploy your own finetuned command-r-0824.ipynb
Comment (youran-qi) [reply notebooks/finetuning/Deploy your own finetuned command-r-0824.ipynb]: OK. Added. Thank you!

**Review by CoderHam — APPROVED** [review]

**Comment by kyle-cohere** [comment]
Most recent changes were 12h ago so I'm assuming @youran-qi has had a chance to test this. Merging to unblock publication

**Pull request merged**
